src/celeritas/em/model/BetheHeitlerModel.cu

```diff
@@ -29,7 +29,7 @@ void BetheHeitlerModel::execute(CoreParams const& params,
         this->action_id(),
         InteractionApplier{BetheHeitlerExecutor{this->device_ref()}});
     static ActionLauncher<decltype(execute)> const launch_kernel(*this);
-    launch_kernel(state, execute);
+    launch_kernel(state, params, TrackOrder::sort_step_limit_action, execute);
```

Comment:
We sort on `ActionOrder::sort_pre` when sorting by along_step Action and `ActionOrder::sort_along` when sorting by step_limit action. Though I think we should add am `ActionOrder::sort_pre_post` as `DiscreteSelect` can update the step_limit action.

Reply:
In any case, we have to update the signature of either the constructor or call operator to pass the params (or just the track order) since the `ActionLauncher` needs to know if Celeritas was configured to sort tracks as we can't reduce the grid size if not.

We can then have the `ActionLauncher` select the correct kernel call if the action order matches the track order Celeritas was configured with , essentially matching [this](https://github.com/celeritas-project/celeritas/blob/e64bff8ed36e0bcc73e60eca56aae8b91e64f933/src/celeritas/track/SortTracksAction.cc#L52) switch case. If we do that we should probably have a single source of truth for the `TrackOrder` <--> `ActionOrder` mapping.